src/isomorphic/React.js

```diff
@@ -129,8 +135,10 @@ if (__DEV__) {
         lowPriorityWarning(
           false,
           'Accessing factories like React.DOM.%s has been deprecated ' +
-            'and will be removed in the future. Use the ' +
-            'react-addons-dom-factories package instead.',
+            'and will be removed in v16.0+. Use the ' +
+            'react-addons-dom-factories package instead. ' +
```

Comment:
I don't think this matches the package name.

https://github.com/facebook/react/tree/master/packages/react-dom-factories

We should probably avoid calling it an "addon" since it was never exposed on `React.addons`.